test/e2e/network/netpol/model.go

```diff
@@ -41,6 +41,29 @@ type Model struct {
 	DNSDomain      string
 }
 
+// NewWindowsModel returns a model specific to windows testing.
+func NewWindowsModel(namespaces []string, podNames []string, ports []int32, dnsDomain string) *Model {
+	return NewModel(namespaces, podNames, ports, []v1.Protocol{v1.ProtocolTCP}, dnsDomain)
+}
+
+// GetProbeTimeoutSeconds returns a timeout for how long the probe should work before failing a check, and takes windows heuristics into account, where requests can take longer sometimes.
+func (m *Model) GetProbeTimeoutSeconds() int {
+	timeoutSeconds := 1
```

Comment:
we should have timeout be in Duration so the suffix is not needed and convert on usage with .Second()

Reply:
i considered that, but the issue winds up being that theres a very nasty behaviour with the probe function, wherein actually if you do anything *other* then seconds iirc, it hangs *forever* which causes all the tests to fail.
So maybe we can consider this in a follow on PR.

Reply:
https://github.com/kubernetes/kubernetes/issues/102334 <-- i think this is worth thinking about / batching w/ other polish offs, if that works for you @bowei  ? Also a good first issue i think for someone to tackle and look at holistically w/ the struct thing